lib/utils.js

```diff
@@ -103,22 +113,21 @@ const processOption = (options, config, hasWhitespace) => {
 }
 
 // Saving into Config Files
-const saveConfig = (config, configPath, callback) => {
+function saveConfig(config, configPath, callback) {
   return properties.stringify(config, {
     path: configPath
   }, callback)
 }
 
 // Custom JSON Stringifier
-const stringify = (obj) => {
-
+// TODO(darinspivey) Why do we need this?  We shouldn't roll our own stringifier.
```

Comment:
@darinspivey, the intention was to properly show the configuration in the command line - well-aligned, clean, etc. - we will need to wipe this out, most probably, in the config work.

Reply:
I get it, but that goes back to programming for human eyes when we should first write code for computers.  This kind of custom logic tends to stick around and become a burden.  We'll look closer at it in the future in case I'm missing something.  I didn't look hard at it, it just feels weird to re-invent that wheel.